src/TestFramework/TestFramework/Attributes/Lifecycle/Cleanup/ClassCleanupExecutionAttribute.cs

```diff
@@ -14,7 +14,7 @@ public class ClassCleanupExecutionAttribute : Attribute
     /// <summary>
     /// Default class cleanup execution.
     /// </summary>
-    public static readonly ClassCleanupBehavior DefaultClassCleanupLifecycle = ClassCleanupBehavior.EndOfAssembly;
+    public static readonly ClassCleanupBehavior DefaultClassCleanupLifecycle = ClassCleanupBehavior.EndOfClass;
```

Comment:
Could you check in AzDo and GitHub if you see some usage of this field? It seems to be unused inside our framework so maybe it could be removed.